app/controllers/admins_controller.rb

```diff
@@ -40,6 +41,29 @@ def index
     @pagy, @users = pagy(user_list)
   end
 
+  # GET /admins/site_settings
+  def site_settings
+  end
+
+  # GET /admins/server_recordings
+  def server_recordings
+    server_rooms = []
```

Comment:
Would it be easier to just do Room.includes(:owner).where(users: { provider: 'user1' }).pluck(:bbb_id) for multitenant and for single tenant just not pass a meeting id in the getRecordings call?

Reply:
Fixed